code/modules/projectiles/projectile/energy/thermal.dm

```diff
@@ -0,0 +1,43 @@
+/obj/projectile/energy/inferno
+	name = "molten nanite bullet"
+	icon_state = "infernoshot"
+	damage = 20
+	damage_type = BURN
+	flag = ENERGY
+	armour_penetration = 10
+	reflectable = NONE
```

Comment:
or you could just make these a bullet subtype so that weird stuff doesn't happen down the line